cookbooks/vankeken_subduction/doc/vankeken_subduction.md

```diff
@@ -0,0 +1,24 @@
+# Van Keken 2008 corner flow recreation
+
+*This section was contributed by Daniel Douglas and Max Rudolph*.
+
+This cookbook recreates a 2D corner flow style subduction model as outlined in van Keken 2008.
+Since ASPECT uses a rectangular mesh by default, the geometry of this model combined with the
+way that the velocities are constrained make using ASPECT's default mesh extremely unstable when
+trying to solve for a continuous pressure field. To circumvent this issue, a custom mesh using
+quadrilaterals was designed that allows for a q2q1 discretization to become stable. The input
+files were modified only slightly from an existing PR #1864 made by Max Rudolph. Further efforts
```

Comment:
Remove the reference to the old PR and its author here. Documentation should document the current state of a software, not its genesis or previous versions.